src/libraries/System.Net.Ping/src/System/Net/NetworkInformation/Ping.PingUtility.cs

```diff
@@ -133,9 +135,11 @@ private static bool TryParseTtlExceeded(string stdout, out PingReply? reply)
                 return false;
             }
 
-            // look for address in "From 172.21.64.1 icmp_seq=1 Time to live exceeded"
+            // look for address in:
+            // - "From 172.21.64.1 icmp_seq=1 Time to live exceeded"
+            // - "From 172.21.64.1: icmp_seq=1 Time to live exceeded"
             int addressStart = stdout.IndexOf("From ", StringComparison.Ordinal) + 5;
-            int addressLength = stdout.IndexOf(' ', Math.Max(addressStart, 0)) - addressStart;
+            int addressLength = stdout.IndexOfAny(s_ttlExceededAddressSeparators, Math.Max(addressStart, 0)) - addressStart;
```

Comment:
Could this use Span or ROSpan instead of extra allocation?

Reply:
Yes, thanks for the suggestion.